cli/command/image/prune.go

```diff
@@ -42,6 +43,7 @@ func NewPruneCommand(dockerCli *command.DockerCli) *cobra.Command {
 	flags := cmd.Flags()
 	flags.BoolVarP(&opts.force, "force", "f", false, "Do not prompt for confirmation")
 	flags.BoolVarP(&opts.all, "all", "a", false, "Remove all unused images, not just dangling ones")
+	flags.BoolVarP(&opts.keepDownloadCache, "keep-download-cache", "k", false, "Don't clean out the download cache")
```

Comment:
This should be on by default... this flag should probably be `disable-download-cache` and revert the daemon to the current behavior, which only saves active downloads.